spec/museum_spec.rb

```diff
@@ -0,0 +1,91 @@
+require "rspec"
+require "./lib/exhibit"
+require "./lib/patron"
+require "./lib/museum"
+
+describe Museum do
+  it 'exists' do
+    dmns = Museum.new("Denver Museum of Nature and Science")
+    expect(dmns).to be_an_instance_of(Museum)
+  end
+
+  it 'attributes' do
+    dmns = Museum.new("Denver Museum of Nature and Science")
+    expect(dmns.name).to eq("Denver Museum of Nature and Science")
+    expect(dmns.exhibits).to eq([])
+  end
+
+  it '#add_exhibit' do
+    dmns = Museum.new("Denver Museum of Nature and Science")
+    gems_and_minerals = Exhibit.new({name: "Gems and Minerals", cost: 0})
+    dead_sea_scrolls = Exhibit.new({name: "Dead Sea Scrolls", cost: 10})
+    imax = Exhibit.new({name: "IMAX",cost: 15})
+    dmns.add_exhibit(gems_and_minerals)
+    dmns.add_exhibit(dead_sea_scrolls)
+    dmns.add_exhibit(imax)
+    expect(dmns.exhibits).to eq([gems_and_minerals, dead_sea_scrolls, imax])
+  end
+
+  it '#recommend_exhibits' do
+    dmns = Museum.new("Denver Museum of Nature and Science")
+    gems_and_minerals = Exhibit.new({name: "Gems and Minerals", cost: 0})
+    dead_sea_scrolls = Exhibit.new({name: "Dead Sea Scrolls", cost: 10})
+    imax = Exhibit.new({name: "IMAX",cost: 15})
+    patron_1 = Patron.new("Bob", 20)
+    patron_1.add_interest("Dead Sea Scrolls")
+    patron_1.add_interest("Gems and Minerals")
+    patron_2 = Patron.new("Sally", 20)
+    patron_2.add_interest("IMAX")
+    dmns.add_exhibit("Gems and Minerals")
+    dmns.add_exhibit("Dead Sea Scrolls")
+    dmns.add_exhibit("IMAX")
```

Comment:
here you should be adding the instances that you created in lines 31-33:

```suggestion
    dmns.add_exhibit(gems_and_minerals)
    dmns.add_exhibit(dead_sea_scrolls)
    dmns.add_exhibit(imax)
```

as `recommend_exhibits` should be returning an array of exhibit objects. This likely messed up how you wrote the `recommend_exhibits` method